Fix flags passed to lint library build to include all needed -I options

diff -urp -x '*~' -x '*.orig' configure.ac configure.ac
--- configure.ac	2008-03-17 20:00:43.000000000 -0700
+++ configure.ac	2009-05-09 16:58:35.161130000 -0700
@@ -50,7 +50,6 @@ XORG_RELEASE_VERSION
 dnl Allow checking code with lint, sparse, etc.
 XORG_WITH_LINT
 XORG_LINT_LIBRARY([Xv])
-LINT_FLAGS="${LINT_FLAGS} ${XV_CFLAGS}"
 
 AC_OUTPUT([Makefile
            include/Makefile
diff -urp -x '*~' -x '*.orig' src/Makefile.am src/Makefile.am
--- src/Makefile.am	2006-09-18 15:54:54.000000000 -0700
+++ src/Makefile.am	2009-05-09 16:58:35.161447000 -0700
@@ -5,7 +5,7 @@ libXv_la_SOURCES = \
          Xvlibint.h
 
 libXv_la_LIBADD = @XV_LIBS@
-AM_CFLAGS = @XV_CFLAGS@ -I$(top_srcdir)/include @MALLOC_ZERO_CFLAGS@
+AM_CPPFLAGS = @XV_CFLAGS@ -I$(top_srcdir)/include @MALLOC_ZERO_CFLAGS@
 
 libXv_la_LDFLAGS = -version-number 1:0:0 -no-undefined
 
